docs/en/samples/hostpath.md

```diff
@@ -0,0 +1,235 @@
+        quota: 50Gi
+        high: "0.99"
+        low: "0.8"
+EOF
+```
+
+> **NOTE:**
+>
+> - `mounts.mountPoint`通过`local://`的前缀来指明要挂载的主机目录(e.g. `/mnt/nfs-imagenet`)
+> - `spec.replicas`和dawnbench测试的worker数量保持一致。比如：单机八卡为1，四机八卡为4
+> - `nodeSelectorTerms`作用是限制在有V100显卡的机器上部署数据集，此处应根据实验环境具体调节
+
+```
+$ kubectl create -f dataset.yaml
+```
+
+3. 检查部署情况
+
+```
+$ kubectl get pv,pvc
+NAME                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS   REASON   AGE
+persistentvolume/imagenet   100Gi      RWX            Retain           Bound    default/imagenet                           3h28m
+
+NAME                             STATUS   VOLUME     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+persistentvolumeclaim/imagenet   Bound    imagenet   100Gi      RWX                           3h28m
+```
+
+### dawnbench
+
+#### 单机八卡
+
+```
+arena submit mpi \
+--name horovod-v2-nfs-fluid-1x8-093009 \
+--gpus=8 \
+--workers=1 \
+--working-dir=/horovod-demo/tensorflow-demo/ \
+--data imagenet:/data \
+-e DATA_DIR=/data/imagenet/imagenet \
+-e num_batch=1000 \
+-e datasets_num_private_threads=8 \
+--image=registry.cn-hangzhou.aliyuncs.com/tensorflow-samples/horovod-benchmark-dawnbench-v2:0.18.1-tf1.14.0-torch1.2.0-mxnet1.5.0-py3.6 \
+./launch-example.sh 1 8
+```
+
+#### 四机八卡
+
+```
+arena submit mpi \
+--name horovod-v2-nfs-fluid-4x8-092910 \
+--gpus=8 \
+--workers=4 \
+--working-dir=/horovod-demo/tensorflow-demo/ \
+--data imagenet:/data \
+-e DATA_DIR=/data/imagenet/imagenet \
+-e num_batch=1000 \
+-e datasets_num_private_threads=8 \
+--image=registry.cn-hangzhou.aliyuncs.com/tensorflow-samples/horovod-benchmark-dawnbench-v2:0.18.1-tf1.14.0-torch1.2.0-mxnet1.5.0-py3.6 \
+./launch-example.sh 4 8
+```
+
+
+
+## Experiment Results
+
+### horovod-1x8
+
+|                         | nfs-hostpath | fluid (cold) | fluid (warm) |
+| ----------------------- | -------- | ------------ | ------------ |
+| Training time                 | 4h20m36s | 4h21m56s     | 4h2m16s     |
+| Speed at the 1000 step(images/second) | 2426.4  | 2467.2      | 8959.7      |
+| Speed at the last step(images/second) | 8218.1   | 8219.8      | 8275.8      |
+| steps                   | 56300    | 56300       | 56300       |
+| Accuracy @ 5            | 0.9280   | 0.9288       | 0.9291      |
+
+### horovod-4x8
+
+|                         | nfs-hostpath | fluid (cold) | fluid (warm) |
+| ----------------------- | ---------- | ------------ | ------------ |
+| Training time                  | 2h9m21s   | 1h40m15s     | 1h29m55s     |
+| Speed at the 1000 step(images/second) | 3219.2    | 11067.2      | 21951.3      |
+| Speed at the last step(images/second) | 15855.7   | 20964.4      | 21869.8      |
+| steps                   | 14070      | 14070        | 14070        |
+| Accuracy @ 5            | 0.9227     | 0.9232       | 0.9228       |
+
+## 结果分析
+
+
+From the test results, the Fluid acceleration effect on 1x8 has no obvious effect,
```

Comment:
From the test results, the Fluid acceleration on 1x8 has no obvious enhancement,  While in the improvements of 4x8, the effect is very obvious.

Reply:
I have a new commit